scripts/install-prerequisites.sh

```diff
@@ -13,5 +13,6 @@ sudo apt install gcc gcc-multilib g++-multilib ninja-build \
     libpng-dev:i386 libjpeg-dev:i386 libfreetype6-dev:i386 \
     ruby-full gcovr cmake  python3 pngquant libinput-dev libxkbcommon-dev \
     libdrm-dev pkg-config wayland-protocols libwayland-dev libwayland-bin \
-    libwayland-dev:i386 libxkbcommon-dev:i386
+    libwayland-dev:i386 libxkbcommon-dev:i386 astyle
```

Comment:
astyle should be installed with[ install_astyle.sh](https://github.com/lvgl/lvgl/blob/master/scripts/install_astyle.sh).